src/commands/do-post.ts

```diff
@@ -36,6 +36,16 @@ export const doPost = async () => {
     let url = ''
     let method = 'GET'
     let body = []
+
+    if (lines[0]) {
```

Comment:
I wonder if this should be in the loop in the `for` loop. If it will, we can write something like:

```http
Content-Type: application/json
POST https://my-api

{
  "hello": "world"
}
```

Reply:
I thank it should in the `for` loop.